src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/Indexing.scala

```diff
@@ -93,12 +95,20 @@ private class Indexing(
   }
 
   private def createElasticSearchView(implicit project: Project, s: Subject): Task[Either[Rejection, Resource]] = {
-    val view: View = ElasticSearchView.default(project.ref)
+    val view = ElasticSearchView.default(project.ref)
     asJson(view).flatMap { json =>
-      val created = resources.create(Id(project.ref, view.id), viewRef, json).value
-      created.mapRetry(
-        createdOrExists,
-        InternalError(s"Couldn't create default ElasticSearch view for project '${project.ref}'"): KgError)
+      view
+        .createIndex[Task]
+        .recover {
+          case err =>
+            logger.error("There was a problem while creating index for default ElasticSearch view", err)
+            ()
+        } *> resources
```

Comment:
Use `>>` or `.flatMap` instead of `*>` since this latest does not have a requirement on which executes first.